apps/explorer/src/components/module/ModulesWrapper.tsx

```diff
@@ -56,7 +56,12 @@ function ModuleViewWrapper({ id, data }: Props) {
                             index < modulesPageNumber * numOfMudulesToShow
                     )
                     .map(([name, code], idx) => (
-                        <ModuleView key={idx} id={id} name={name} code={code} />
+                        <div key={idx}>
```

Comment:
I know this is partially just from the old code but can the key be the name instead of the index?